### PR TITLE
R10/stable v4.2/rv/feature rpm indicator v4.3

### PR DESCRIPTION
**Pull request by rajendra-ra**
- Add RPM indicator

### CODE REVIEW

**Comment by akshar-ra** [comment]
Please follow the git branch naming convention

File: qgroundcontrol.qrc [hunk 1]
```diff
@@ -17,6 +17,8 @@
         <file alias="ROIIndicator.qml">src/ui/toolbar/ROIIndicator.qml</file>
         <file alias="TelemetryRSSIIndicator.qml">src/ui/toolbar/TelemetryRSSIIndicator.qml</file>
         <file alias="VTOLModeIndicator.qml">src/ui/toolbar/VTOLModeIndicator.qml</file>
+        <file alias="FuelIndicator.qml">src/ui/toolbar/FuelIndicator.qml</file>
```
Comment (akshar-ra) [inline qgroundcontrol.qrc:20]: These changes are not included in the documentation

File: src/Vehicle/Vehicle.cc [hunk 2]
```diff
@@ -96,6 +96,10 @@ const char* Vehicle::_distanceToGCSFactName =       "distanceToGCS";
 const char* Vehicle::_hobbsFactName =               "hobbs";
 const char* Vehicle::_throttlePctFactName =         "throttlePct";
 
+//const char*  Vehicle::_fuelLevelFactName =          "fuelLevel";
+const char*  Vehicle::_engineRPMFactName =          "engineRPM";
+const char*  Vehicle::_rotorRPMFactName =           "rotorRPM";
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:101]: These changes are not included in the documentation

File: src/Vehicle/Vehicle.cc [hunk 3]
```diff
@@ -1042,6 +1060,20 @@ void Vehicle::_handleAttitudeWorker(double rollRadians, double pitchRadians, dou
     _headingFact.setRawValue(yaw);
 }
 
+//void Vehicle::_handleFuelLevel(mavlink_message_t& message)
+//{
+//    mavlink_ status;
+//    mavlink_msg_router_channel_status_decode(&message, &status);
+//    _routerStatusFact.setRawValue(status.channel_active);
+//}
+void Vehicle::_handleRPM(mavlink_message_t& message)
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:1069]: Please add comment to the function

File: src/ui/toolbar/FuelIndicator.qml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
```
Comment (akshar-ra) [inline src/ui/toolbar/FuelIndicator.qml:8]: please add comment on the top describing the use of this file

File: src/ui/toolbar/RPMIndicator.qml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
```
Comment (akshar-ra) [inline src/ui/toolbar/RPMIndicator.qml:7]: please add comment on the top describing the use of this file

File: qgroundcontrol.qrc (hunk 1 above)
Comment (rajendra-ra) [reply qgroundcontrol.qrc:20]:
```suggestion
```
fuel indicator is not implemented separately imstead uses battery status indicator.
this `FuelIndicator.qml` is not required.

File: src/ui/toolbar/FuelIndicator.qml (hunk 4 above)
Comment (rajendra-ra) [reply src/ui/toolbar/FuelIndicator.qml:8]: this file is not used

File: src/ui/toolbar/RPMIndicator.qml (hunk 5 above)
Comment (rajendra-ra) [reply src/ui/toolbar/RPMIndicator.qml:7]: comment added #8

File: src/Vehicle/Vehicle.cc (hunk 3 above)
Comment (rajendra-ra) [reply src/Vehicle/Vehicle.cc:1069]: commented #8

File: src/Vehicle/Vehicle.cc (hunk 2 above)
Comment (rajendra-ra) [reply src/Vehicle/Vehicle.cc:101]: added in doc

**Pull request closed without merging**